tgui/packages/tgui/interfaces/ChemiCompiler/ChemiCompilerMemory.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/Garash2k)
+ * @license ISC
+ */
+
+import { useBackend } from '../../backend';
+import { ChemiCompilerData } from './type';
+import { Button, Section, Stack } from '../../components';
+
+export const ChemiCompilerMemory = (_props, context) => {
+  const { act, data } = useBackend<ChemiCompilerData>(context);
+  const { buttons, sx, tx, ax, theme } = data;
+  return (
+    <Section title="Memory">
+      sx: <em>{sx}</em> -
+      ax: <em>{tx}</em> -
+      tx: <em>{ax}</em>
+      <Stack wrap>
+        {buttons.map((button, index) => (
+          <Stack.Item key={index} style={{ border: `1px solid ${theme === "syndicate" ? "#397439" : "#88bfff"}`, "border-radius": "0.2rem" }} p={1} m={1}>
```

Comment:
Rather than styling the `Stack.Item` consider putting a `Section` inside it with the `M#` as titles. I think that should have decent default styling in this use case. No issue with buttons as content for these, I think that works here.